pyproject.toml

```diff
@@ -9,7 +9,7 @@ build-backend = "hatchling.build"
 [project]
 name = "jupyter-fs"
 description = "A Filesystem-like mult-contents manager backend for Jupyter"
-version = "0.4.1"
+version = "1.0.0"
```

Comment:
Could be dynamic? https://hatch.pypa.io/latest/version/

Reply:
Easier to manage through bumpversion imo, it will just find/replace everywhere